packages/sdk-js/src/handlers/auth.ts

```diff
@@ -62,6 +62,10 @@ export class AuthHandler {
 
 		if (this.mode === 'json') {
 			const refreshToken = await this.storage.getItem('directus_refresh_token');
+			if (!refreshToken){
+				return false;
+			}
+			this.storage.removeItem('directus_refresh_token');
```

Comment:
There is no need to remove this, it will be replaced when new refresh token is recieved. I'm already working on preventing sending request if user is not logged in